modules/sdk/project-templates/project-templates-activator/bnd.bnd

```diff
@@ -1,7 +1,7 @@
 Bundle-Description: Creates a Liferay module project that customizes the starting and stopping of a Liferay bundle.
 Bundle-Name: Liferay Project Templates Activator
 Bundle-SymbolicName: com.liferay.project.templates.activator
-Bundle-Version: 1.0.28
+Bundle-Version: 1.0.29
```

Comment:
Don't change the bundle versions in this commit.  This change should always be in a separate commit for updating latest versions.